matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/send/LocalEchoEventFactory.kt

```diff
@@ -347,14 +347,21 @@ internal class LocalEchoEventFactory @Inject constructor(
     fun createMediaEvent(
             roomId: String,
             attachment: ContentAttachmentData,
-            rootThreadEventId: String?
+            rootThreadEventId: String?,
```

Comment:
Not blocking but I am wondering if we should remove the `rootThreadEventId` param and build the `relatesTo` param in case of threads at an upper level?

Reply:
I started considering this point but I am not sure it would make sense to build the `relatesTo` related to a thread in an upper level, IMO it makes sense to do it in the factory 😕